config/default.yml

```diff
@@ -224,3 +228,7 @@ RSpec/VerifiedDoubles:
   Description: Prefer using verifying doubles over normal doubles.
   Enabled: true
   IgnoreSymbolicNames: false
+
+FactoryGirl/DynamicAttributeDefinedStatically:
+  Description: Prefer to declare dynamic attribute values in a block.
```

Comment:
Minor wording tweak: I might start the sentence with "Prefer declaring" or just "Declare". Not a requirement, just a suggestion.